.github/workflows/platform-deployment.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
   deploy_agreement_management:
     name: Agreement Management
     needs: [deploy_configs]
-    if: ${{ inputs.environment != 'dev-refactor' && inputs.environment != 'test' && inputs.environment != 'prod'}}
+    if: ${{ inputs.environment != 'dev-refactor' && inputs.environment != 'qa' &&inputs.environment != 'test' && inputs.environment != 'prod'}}
```

Comment:
Needs changes from [this PR](https://github.com/pagopa/pdnd-interop-platform-deployment/pull/568/files#diff-ff999ffa8e4856926b1bf4c0ac438ec807954605b8102afd957c084acb79806bR92) as well, maybe after that PR has been merged